commands/apps/errors.js

```diff
@@ -40,6 +40,13 @@ function * run (context, heroku) {
       host: 'api.metrics.herokai.com',
       path: `/apps/${context.app}/router-metrics/errors?${DATE}&process_type=web`,
       headers: {Range: ''}
+    }).catch((err) => {
+      if (err.statusCode === 400 && err.body && err.body.message &&
+          err.body.message.startsWith('invalid process_type provided')
```

Comment:
@djcp make case insensitive in https://github.com/heroku/heroku-apps/pull/136/commits/618d40b6193ac2d005af73ba36a1202f559440b6